test/501_migrations/one_migration.t

```diff
@@ -4,1127 +4,159 @@ test checks whether parsing on 5.0.0 (result of test running on 5.0.0) is the sa
-            (pmod_attributes ()))))
-         (pmod_loc
-          ((loc_start
-            ((pos_fname file.ml) (pos_lnum 1) (pos_bol 0) (pos_cnum 9)))
-           (loc_end
-            ((pos_fname file.ml) (pos_lnum 1) (pos_bol 0) (pos_cnum 24)))
-           (loc_ghost false)))
-         (pmod_attributes ())))
-       (pmb_attributes ())
-       (pmb_loc
-        ((loc_start
-          ((pos_fname file.ml) (pos_lnum 1) (pos_bol 0) (pos_cnum 0)))
-         (loc_end ((pos_fname file.ml) (pos_lnum 1) (pos_bol 0) (pos_cnum 24)))
-         (loc_ghost false))))))
-    (pstr_loc
-     ((loc_start ((pos_fname file.ml) (pos_lnum 1) (pos_bol 0) (pos_cnum 0)))
-      (loc_end ((pos_fname file.ml) (pos_lnum 1) (pos_bol 0) (pos_cnum 24)))
-      (loc_ghost false))))
-   ((pstr_desc
-     (Pstr_module
-      ((pmb_name
-        ((txt (M))
-         (loc
-          ((loc_start
-            ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 32)))
-           (loc_end
-            ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 33)))
-           (loc_ghost false)))))
-       (pmb_expr
-        ((pmod_desc
-          (Pmod_apply
-           ((pmod_desc
-             (Pmod_ident
-              ((txt (Lident F))
-               (loc
-                ((loc_start
-                  ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 36)))
-                 (loc_end
-                  ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 37)))
-                 (loc_ghost false))))))
-            (pmod_loc
-             ((loc_start
-               ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 36)))
-              (loc_end
-               ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 37)))
-              (loc_ghost false)))
-            (pmod_attributes ()))
-           ((pmod_desc (Pmod_structure ()))
-            (pmod_loc
-             ((loc_start
-               ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 36)))
-              (loc_end
-               ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 40)))
-              (loc_ghost false)))
-            (pmod_attributes ()))))
-         (pmod_loc
-          ((loc_start
-            ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 36)))
-           (loc_end
-            ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 40)))
-           (loc_ghost false)))
-         (pmod_attributes ())))
-       (pmb_attributes ())
-       (pmb_loc
-        ((loc_start
-          ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 25)))
-         (loc_end
-          ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 40)))
-         (loc_ghost false))))))
-    (pstr_loc
-     ((loc_start ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 25)))
-      (loc_end ((pos_fname file.ml) (pos_lnum 2) (pos_bol 25) (pos_cnum 40)))
-      (loc_ghost false)))))
+  $ ./identity_driver.exe -pp-simple file.ml
+  [ Pstr_attribute
```

Comment:
Ah that's good to know, I didn't think about floating attributes! I'll open a PR to fix that!